test/e2e/serial/workload_placement_test.go

```diff
@@ -507,6 +518,135 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, schedulerName)
 		})
 	})
+
+	When("cluster has two feasible nodes with taint but only one has the requested resources on a single NUMA zone", func() {
+		timeout := 5 * time.Minute
+		BeforeEach(func() {
```

Comment:
it seems we're not using `e2efixture`, is it the case? if so, why?

Reply:
Why not? we do, check l line 535 we're using the ```fxt.Client```

Reply:
right, my bad. Reading to hastily. It seems good.